components/BB.js

```diff
@@ -88,7 +36,6 @@ export class BB extends Component {
 
   constructor(props) {
     super(props);
-    this.nextStepsRef = React.createRef(); // create a ref object
```

Comment:
This was not actually doing anything since we started scrolling to elements based on id when the scroll offset on line 121 was added.